docs/crashlytics/usage.mdx

```diff
@@ -3,15 +3,41 @@ title: Using Firebase Crashlytics
 sidebar_label: Usage
 ---
 
+## Getting started
+
+Enable Firebase Crashytlics in the Firebase console.
+
+![Crashytlics - Enable in the console](enable-firebase-crashlytics.png)
```

Comment:
I tried running this locally and got this error. I guess the path needs updating.
<img width="946" alt="Screenshot 2021-03-22 at 17 47 28" src="https://user-images.githubusercontent.com/16018629/112034716-b076c280-8b36-11eb-8e37-899819c42d95.png">